sleuth_backend/solr/connection.py

```diff
@@ -52,18 +58,49 @@ def fetch_core_schema(self, name):
         return response['schema']
 
     def insert_document(self, core, doc):
```

Comment:
Can we change this function name to be `queue_document` to differentiate it from `insert_documents`, since it queues.